ports/stm32/boards/NUCLEO_G474RE/mpconfigboard.mk

```diff
@@ -0,0 +1,21 @@
+DEBUG = 0
+
+# MCU settings
+MCU_SERIES = g4
+CMSIS_MCU = STM32G474xx
+MICROPY_FLOAT_IMPL = double
+AF_FILE = boards/stm32g474_af.csv
+LD_FILES = boards/stm32g474.ld boards/common_basic.ld
+
+# MicroPython settings
+#MICROPY_PY_LWIP = 0  # Geen ETH
+#MICROPY_PY_USSL = 0
+#MICROPY_SSL_MBEDTLS = 0
+#MICROPY_PY_UASYNCIO = 0
+#MICROPY_PY_UZLIB = 0
+#MICROPY_PY_UJSON = 0
+#MICROPY_PY_URE = 0
+#MICROPY_PY_FRAMEBUF = 0
+#MICROPY_PY_USOCKET = 0
+#MICROPY_PY_NETWORK = 0
```

Comment:
These commented out items can all be removed (and most of these don't work at the linker level).

Reply:
Removed all the comments.